venues/ICLR.cc/2020/Conference/python/buddyAcCommentProcess.js

```diff
@@ -0,0 +1,47 @@
+function(){
+    var or3client = lib.or3client;
+
+    var CONFERENCE_ID = 'ICLR.cc/2020/Conference';
+    var SHORT_PHRASE = 'ICLR 2020';
+
+    or3client.or3request(or3client.notesUrl + '?id=' + note.forum, {}, 'GET', token)
+    .then(function(result) {
+
+      var forumNote = result.notes[0];
+      var AREA_CHAIRS_ID = CONFERENCE_ID + '/Paper' + forumNote.number + '/Area_Chairs';
```

Comment:
it seems this group contains both area chairs, are we sending the emails to both?

Reply:
Thanks for noting this. I have corrected it.